doc/nrf/includes/nrf_cloud_rest_sample_requirements.txt

```diff
@@ -0,0 +1,38 @@
+.. requirement_keysign_start
+
+This sample uses the `nRF Cloud REST API`_, which requires that your device has a valid signing key installed and registered with `nRF Cloud`_.
+
+.. requirement_keysign_moreinfo_start
+
+(See `nRF Connect for Cloud REST API`_ and `Securely Generating Credentials on the nRF9160`_ for more details on this requirement).
+
+Modem version v1.3.x or later is also required.
+
+* Your device may ship with modem firmware older than v1.3.x, so verify that you have the latest installed.
+* Instructions for updating your device's modem firmware may be found here:
+
+  * `Updating modem firmware on nRF9160 DK <updating nRF9160 DK modem firmware>`_
+  * `Updating modem firmware on Thingy:91 <updating Thingy:91 modem firmware>`_
+
+
+.. requirement_cert_provision_start
+
+To obtain and register a valid signing key, you may:
+
+* Provision your device on nRF Cloud using Just-In-Time Provisioning (JITP) (detailed below).
+* Provision your device on nRF Cloud with preconnect provisioning (detailed in `nRF Cloud Provisioning`_).
+* Install/generate a private key on your device and register its public key with nRF Cloud (detailed in `Securely Generating Credentials on the nRF9160`_).
+
+To provision your device on nRF Cloud using JITP:
+
+1) :ref:`Ensure valid nRF Cloud credentials are installed <nrf9160_ug_updating_cloud_certificate>`.
+   nRF9160 DK and Thingy:91 have credentials pre-installed, but some may be shipped with RSA256 credentials; JWT signging requires ES256 credentials.
+   These devices will need new credentials installed before they can perform REST requests.
+
+#) Install a sample or application that supports JITP.
+   The :ref:`Asset Tracker V2 <asset_tracker_v2>` application may be used for this purpose.
+
+#) Allow your device to connect to nRF Cloud and add it to the nRF Cloud portal.
+   Your device should appear as connected in the devices list of the nRF Cloud portal.
```

Comment:
"should appear as connected" - this is only true if the device is using MQTT (ATv2)
you could just say the device will appear in the device list